packages/mdctl-axon-tools/lib/StudyManifestTools.js

```diff
@@ -669,18 +729,95 @@ class StudyManifestTools {
           oracleQuestions = await this.getExportObjects(org, 'orac__form_questions', null, orgReferenceProps),
           oracleEvents = await this.getExportObjects(org, 'orac__events', null, orgReferenceProps)
 
-
     return [
-      ...tasks, ...steps, ...branches,
-      ...visitSchedules, ...visits, ...groups, ...faults, ...reports, ...sites,
-      ...groupTasks, ...taskAssignments, ...participantSchedules, ...anchorDateTemplates,
-      ...patientFlags, ...documentTemplates, ...knowledgeChecks, ...defaultDoc,
-      ...lookerIntegrationRecords,
+      ...tasksAndDependencies, ...visitSchedulesAndDependencies, ...faults, ...reports, ...sites,
+      ...groupsAndDependencies, ...taskAssignments, ...participantSchedules, ...anchorDateTemplates,
+      ...patientFlags, ...templatesAndDependencies, ...lookerIntegrationRecords,
       ...vendorIntegrationRecords, ...integrationMappings, ...integrationPipelines,
-      ...oracleStudies, ...oracleSites,
-      ...oracleForms, ...oracleQuestions, ...oracleEvents
+      ...oracleStudies, ...oracleSites, ...oracleForms, ...oracleQuestions, ...oracleEvents
     ]
   }
+  */
+
+  async getStudyManifestEntities(org, study, manifestObject, orgReferenceProps) {
+    const manifestEntities = [],
+          // Define the available entities to export or get them from the manifest in input
+          manifestKeys = (study)
```

Comment:
By applying the refactors mentioned [above](https://github.com/Medable/mdctl/pull/379/files#r888988504), this line would be:
```
manifestKeys = study ? this.availableObjectNames : Object.keys(manifestObject)
```